virtual_rainforest/models/animals/scaling_functions.py

```diff
@@ -35,22 +21,18 @@ def damuths_law(
 
     Args:
         mass: The body-mass [kg] of an AnimalCohort.
-        taxa: The taxa category of the animal cohort [toy: mammal or bird].
-        diet: The diet category of the animal cohort [toy: herbivore or carnivore].
-        terms: The dictionary of population density terms used, default to Damuth.
+        terms: The tuple of population density terms used, default to Damuth.
 
     Returns:
         The population density of that AnimalCohort [individuals/km2].
 
     """
-    exp = terms[taxa][diet][0]
-    coef = terms[taxa][diet][1]
+    exp = terms[0]
+    coef = terms[1]
     return ceil(coef * mass**exp)
```

Comment:
There's no real need to extract them to actual variables?

```suggestion
    return ceil(terms[1] * mass**terms[0])
```